README.md

```diff
@@ -102,7 +102,47 @@ A preview of the VDK CLI commands:
 
 
 # Getting Started
-Installing VDK is a simple pip command. See the [Getting Started](https://github.com/vmware/versatile-data-kit/wiki/getting-started) guide to install VDK and create a data job.
+
+## Vdk-core and CLI
+
+```
+pip install vdk-core
+```
+
+This installs the core vdk package and the vdk command line interface. You can
+use them to run jobs in your local shell environment.
+
+## Running with docker, helm, and k8s
+
+**Prerequisites**
+- [helm](https://helm.sh/docs/intro/install)
+- [docker](https://docs.docker.com/get-docker)
+- [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) (version 0.11.1 or later)
+
+```
+pip install quickstart-vdk
+vdk server --install
+```
+
+This installs the necessary python modules and runs the vdk server.
+The server includes:
```

Comment:
I prefer to link to the https://github.com/vmware/versatile-data-kit/wiki/Installation#install-versatile-data-kit-control-service section than repeat this here.